### PR TITLE
[ICDS Dashboard] LS launched based on all LS APP forms

### PR DESCRIPTION
**Pull request by Rohit25negi**
This code change is to make the ls launched based on all form type submission rather than just three forms.

ticket: https://app.asana.com/0/1112385193248823/1145768119172769

### CODE REVIEW

**Comment by Rohit25negi** [comment]
@emord  @calellowitz  @czue  only first comment is the change.  Other two are cleanup

**Comment by czue** [comment]
going to let others with more context review

File: custom/icds_reports/utils/aggregation_helpers/distributed/ls_awc_visit_form.py [hunk 1]
```diff
@@ -29,15 +29,14 @@ def aggregate_query(self):
             state_id,
             location_id as supervisor_id,
             %(start_date)s::DATE AS month,
-            count(*) as form_count,
-            count(*) FILTER (WHERE location_entered is not null and location_entered <> '') as awc_visits
+            count(*) as awc_visits
                 FROM "{ucr_tablename}"
                 WHERE state_id=%(state_id)s AND submitted_on >= %(start_date)s AND
```
Comment (emord) [inline custom/icds_reports/utils/aggregation_helpers/distributed/ls_awc_visit_form.py]: nit. looks like there is some inconsistent indentation here

**Review by emord — COMMENTED** [review]
3 comments are just nits. I don't have any context into the intended behavior, but these seem fine to me. 

Note that when removing columns it does require a lock on the table which could take quite a while if the agg were to be running at the same time.

This is an unused(?) and new table so its probably fine in this class, but something that may need to be split into a different PR in the future

**Comment by Rohit25negi** [comment]
@emord  made the changes. Does it look good to you now?

**Pull request merged**